src/nvim/main.c

```diff
@@ -205,6 +205,8 @@ int main(int argc, char **argv)
   set_vim_var_string(VV_PROGPATH, (char_u *)argv[0], -1);
   set_vim_var_string(VV_PROGNAME, path_tail((char_u *)argv[0]), -1);
 
+  set_vim_var_dict(VV_COMPLETED_ITEM, dict_alloc());
```

Comment:
Just plopping this into main() does not seem like the best approach. Isn't there somewhere more appropriate, like `eval_init`? I'm looking around now.